app/Models/CharacterRace.php

```diff
@@ -44,6 +49,15 @@ public function race(): BelongsTo
 
     public function getCharacterRacesAttribute()
     {
-        return $this->character->races;
+        if (auth()->check() && auth()->user()->isAdmin()) {
```

Comment:
This seems overly complicated. How is it done in the org members view? The model shouldn't have any concept of is a user is an admin or not, that's the policy's job